.travis.yml

```diff
@@ -4,5 +4,3 @@ script:
   - npm test
 node_js:
   - "node"
-  - "6"
```

Comment:
please add an explicit version instead of those 2

Reply:
node uses the latest stable version, we can add "lts/*" which will be the same supported version for every 4 years, but specificying a specific version ourselves seems counter productive